packages/govuk-frontend/src/govuk/components/error-summary/error-summary.mjs

```diff
@@ -1,10 +1,6 @@
-/* eslint-disable es-x/no-function-prototype-bind -- Polyfill imported */
-
 import { mergeConfigs } from '../../common/index.mjs'
 import { normaliseDataset } from '../../common/normalise-dataset.mjs'
 import '../../vendor/polyfills/Element/prototype/closest.mjs'
```

Comment:
Is this polyfill still needed if we're dropping JS support in IE11 as well?

Reply:
Yeah but since it's still a supported browser I've left it in until we pick up `<script type="module">`

Otherwise IE11 will load and run code that will crash